### DISCUSSION TITLE
痛苦

### DISCUSSION THREAD

**Discussion by 332396jjkk** [share121/inter-knot#2149] [其他]
艾莲保底就算了，青衣还歪了，服啦，刚开服一个版本啊，这么非

**Comment by SilverWolf-07L** [comment]
+1辣鸡游戏，不如我在星核猎手玩的那个游戏好玩，走了，回去黑公司内网去了<img src="https://img-static.mihoyo.com/communityweb/upload/76cc94063d9c0efb2335e8487299cd0a.png" alt="银狼 吹泡泡" width="75" >
